ports/mimxrt/machine_rtc.c

```diff
@@ -41,6 +42,13 @@ STATIC const machine_rtc_obj_t machine_rtc_obj = {{&machine_rtc_type}};
 
 // Start the RTC Timer.
 void machine_rtc_start(void) {
+    snvs_hp_rtc_config_t hpconfig;
+    SNVS_HP_RTC_GetDefaultConfig(&hpconfig);
+    SNVS_HP_RTC_Init(SNVS, &hpconfig);
+
+    snvs_lp_srtc_config_t lpconfig;
+    SNVS_LP_SRTC_GetDefaultConfig(&lpconfig);
+    SNVS_LP_SRTC_Init(SNVS, &lpconfig);
```

Comment:
What exactly does this do, does it reset the RTCs?  Does the RTC retain the time if it was already set?

Might be worth adding a comment to explain what is going on.

Reply:
Yes, it retains the RTC time if it was set.
I did not look into the details of the added init. But the effect was, that it re-enables the capability of  RTC to retain the time on battery power when the main power is off. One user in the MP forum and previously users of CP mentioned that.
I can look in to the details and add a comment, if I understand the mechanism,

Reply:
Looking into the details of the Init calls, the do not much. Besides enabling the clock they just clear tamper, enable register access and clear the calibration setting. The latter should not happen, so I removed the Init calls and just add the useful settings. Maybe it was a tamper event that happened to the user's device. I tested that the clock keeps running on battery with power off, and made a test build for ALL mimxrt boards. The PR is updated.

Reply:
Thanks for checking and updating.